shotwell/shotwell-launcher

```diff
@@ -0,0 +1,7 @@
+#! /bin/bash
+if [ ! -z "$1" ]
+  then
+    pwd=$PWD/
+fi
+cd $SNAP
+desktop-launch shotwell "$pwd$@"
```

Comment:
Is the above something we should add to the desktop launcher? @didrocks what do you think?